src/app/components/MostRead/index.tsx

```diff
@@ -39,6 +39,72 @@ interface MostReadProps {
   sendOptimizelyEvents?: boolean;
 }
 
+// We render amp on ONLY STY, CSP and ARTICLE pages using amp-list.
```

Comment:
Looks like this logic may be a bit unnecessary as we only have Article pages now. Maybe we can raise a seperate PR to clean this up or raise a tech ticket?

Reply:
Yeah makes sense, I can raise a tech ticket for this